SRE-0000 ci: pipeline-lib test only - DO NOT MERGE

grom72 opened the pull request:
### Steps for the author:

* [ ] Commit message follows the [guidelines](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Appropriate [Features or Test-tag](https://daosio.atlassian.net/wiki/spaces/DC/pages/10984259629/Test+Tags) pragmas were used.
* [ ] Appropriate [Functional Test Stages](https://daosio.atlassian.net/wiki/spaces/DC/pages/12147556353/CI+Functional+Test+Stages) were run.
* [ ] At least two positive code reviews including at least one code owner from each category referenced in the PR.
* [ ] Testing is complete. If necessary, forced-landing label added and a reason added in a comment.

#### After all prior steps are complete:
* [ ] Gatekeeper requested (daos-gatekeeper added as a reviewer).

grom72 committed:
SRE-0000 : Test for SPDK build

Signed-off-by: Tomasz Gromadzki <tomasz.gromadzki@hpe.com>

daosbuild3 commented:
Test stage Unit Test with memcheck completed with status UNSTABLE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net/job/daos-stack/job/daos//view/change-requests/job/PR-16619/4/testReport/

grom72 committed:
Echo pip config

Signed-off-by: Tomasz Gromadzki <tomasz.gromadzki@hpe.com>

Priority: 2
Cancel-prev-build: false
Skip-python-bandit: true
Skip-unit-test: true
Skip-unit-test-memcheck: true
Skip-func-hw-test: true
Skip-build-el8-gcc: true
Skip-unit-tests:true
Skip-NLT: true
Skip-fault-injection-test: true
Skip-func-vm-all: true

grom72 committed:
Test 2

Signed-off-by: Tomasz Gromadzki <tomasz.gromadzki@hpe.com>

daosbuild3 commented:
Test stage Functional on EL 9 completed with status FAILURE.  https://jenkins-3.daos.hpc.amslabs.hpecorp.net/job/daos-stack/job/daos/job/PR-16619/5/display/redirect
